pocs/core.py

```diff
@@ -223,6 +224,9 @@ def power_down(self):
                 "Shutting down {}, please be patient and allow for exit.".format(
                     self.name))
 
+            if not self.observatory.close_dome():
+                self.logger.critical('Unable to close dome!')
```

Comment:
This is perhaps one of those places where we need to somehow add email support.

Reply:
Makes sense. Perhaps we should extend the idea of the "say" method to include a "yell" method, designed to ask for help.